src/constructs/rds/instance.test.ts

```diff
@@ -80,14 +80,9 @@ describe("The GuDatabaseInstance class", () => {
       devXBackups: { enabled: true },
     });
     console.log(JSON.stringify(stack.tags.tagValues()));
-    GuTemplate.fromStack(stack).hasGuTaggedResource("AWS::RDS::DBInstance", {
-      appIdentity: { app: "testing" },
-      additionalTags: [
-        {
-          Key: "devx-backup-enabled",
-          Value: "true",
-        },
-      ],
+    GuTemplate.fromStack(stack).hasResourceWithTag("AWS::RDS::DBInstance", {
+      Key: "devx-backup-enabled",
```

Comment:
I've manually checked that setting the wrong tag value (or mistyping the key) here causes the test to fail.